pkg/image/importer/importer.go

```diff
@@ -211,6 +219,17 @@ func (i *ImageStreamImporter) importImages(ctx gocontext.Context, retriever Repo
 	}
 }
 
+// isAllowedRegistry returns true if the given registry URL is allowed for the import in
+// the master configuration.
+func (i *ImageStreamImporter) isAllowedRegistry(registry string) bool {
+	for _, allowedRegistry := range i.allowedRegistries {
+		if matched, err := filepath.Match(allowedRegistry, registry); matched && err == nil {
```

Comment:
yeah, I will probably craft something custom and make it part of util/strings.go